data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -117,6 +117,34 @@ datasets:
         field_type: float
         include_in_downloadable_files: true
         include_in_tiles: true
+  - long_name: "Formerly Used Defense Sites"
+    short_name: "FUDS"
+    module_name: "us_army_fuds"
+    load_fields:
+      - short_name: "eligibile_count"
+        df_field_name: "ELIGIBLE_FUDS_COUNT_FIELD_NAME"
+        long_name: "Count of eligible Formerly Used Defense Site (FUDS) properties centroids"
+        description_short: 
+          "The number of FUDS marked as Eligible and Has Project in the tract."
+        field_type: int64
+        include_in_tiles: true
+        include_in_downloadable_files: true
+      - short_name: "ineligibile_count"
```

Comment:
maybe `not_fuds_ct`, same comment as below? As an aside -- since shapefiles can only have column names with 10 or fewer characters, I think we want the "meaning" of the column to be clear as early on in the name as possible?